lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -186,6 +188,18 @@ module Constants
                  :default => nil,
                  :secret => true
 
+    # Whether to collect metrics about the plugin usage. Use configuration
+    # parameter monitoring_type to select a monitoring system you want to use.
```

Comment:
Ah. I felt the first sentence "Whether to collect metrics ..." referred to the `enable_monitoring` config param whereas the second sentence "Use configuration parameter monitoring_type..." is associated with the config param `monitoring_type` . This is a nit comment.